tests/apollo/test_skvbc_restart_recovery.py

```diff
@@ -84,25 +88,31 @@ async def test_restarting_replica_with_client_load(self, bft_network):
         # uncomment for live tracking of log messages from the test
         # log = foo()
 
-        for v in range(300):
+        async def client_load(task_status=trio.TASK_STATUS_IGNORED):
+            with trio.CancelScope() as scope:
+                task_status.started(scope)
+                await skvbc.send_indefinite_ops()
+
+        for v in range(loops * 100):
```

Comment:
So with the current parameters, this would loop for 1600 iterations?

Reply:
Correct

Reply:
It is quite fast.

Reply:
OK, thanks for the clarification.